test/helpers/utils.go

```diff
@@ -497,11 +497,21 @@ func DoesNotRunOnNetNextKernel() bool {
 	return !RunsOnNetNextKernel()
 }
 
+// RunsOn419Kernel checks whether a test case is running on the 4.19 kernel.
+func RunsOn419Kernel() bool {
+	return os.Getenv("KERNEL") == "419"
+}
+
+// DoesNotRunOn419Kernel is the complement function of RunsOn419Kernel.
+func DoesNotRunOn419Kernel() bool {
+	return !RunsOn419Kernel()
+}
```

Comment:
nit: Can't we just open-code the `!RunsOn419Kernel()` in the callers?

Reply:
Yeah, probably. I was following the pattern established by other, similar helpers.